documentation/modules/rn-2.4.adoc

```diff
@@ -0,0 +1,136 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-24_{context}"]
+= {project-full} 2.4
+
+You can migrate virtual machines (VMs) from VMware vSphere or {rhv-full} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-24_{context}"]
+== Technical changes
+
+This release has the following technical changes:
+
+.Faster disk image migration without using virt-v2v
+
+IDisk images are not converted anymore using virt-v2v when migrating from RHV. This way, the migration is faster and not limited to guest operating systems that are supported by virt-v2v. link:https://github.com/kubev2v/forklift-controller/issues/403[(forklift-controller#403)]
+
+.Faster migration using conversion pod disk transfer
+
+vSphere to the local OCP cluster migration uses conversion pod instead of Containerized Data Importer (CDI), accelerating the migration.
```

Comment:
```suggestion
When migrating from vSphere to the local OCP cluster, the conversion pod transfers the disk data instead of Containerized Data Importer (CDI), accelerating the migration.
```